xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -1008,7 +1008,7 @@ static void DisplayReconfigured(CGDirectDisplayID display,
 
     // Blank other displays if requested.
     if (blankOtherDisplays)
-      BlankOtherDisplays(0);
+      BlankOtherDisplays(m_lastDisplayNr);
```

Comment:
did you test this and did it work?
from looking into the code I would think that m_lastDisplayNr is current

Reply:
Yep the latter. And i have tested this - hence saw that it always blanked screen 1 for me - even it 1 was the current ;)